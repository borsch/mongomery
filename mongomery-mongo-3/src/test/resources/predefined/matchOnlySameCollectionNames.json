{
    "col1": [
        {
            "field": 1
        }
    ],
    "col2": [
        {
            "field": "a"
        }
    ],
    "col3": [

    ]
}