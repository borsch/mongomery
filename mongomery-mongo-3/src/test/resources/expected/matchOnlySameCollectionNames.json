{
    "col1": [
        {
            "field": 1
        }
    ]
}